docs/api/app.md

```diff
@@ -778,7 +786,7 @@ Returns `Object`:
 **Note:** This API has no effect on
 [MAS builds][mas-builds].
 
-### `app.setLoginItemSettings(settings)` _macOS_ _Windows_
+### `app.setLoginItemSettings(settings[, path, args])` _macOS_ _Windows_
```

Comment:
I initially tried to bolt them on but ran into issues with the Mate converter. Someone better than me at V8 could probably work it out.